### PR TITLE
Updating Adobe's Source Code Pro and adding Italic styles

### PR DESCRIPTION
**Pull request by mwberry**
Adobe recently released version 2.010 of their regular style and version 1.030 of their italic style [1]. This pull request brings the project up to date with upstream. I have a CLA on file already.

[1] https://github.com/adobe-fonts/source-code-pro/releases/tag/2.010R-ro%2F1.030R-it

### CODE REVIEW

**Comment by rsheeter** [comment]
Thank you for the PR.

File: ofl/sourcecodepro/METADATA.json [hunk 1]
```diff
@@ -68,7 +68,70 @@
       "weight": 600,
       "filename": "SourceCodePro-Semibold.ttf",
       "copyright": "Copyright 2010, 2012 Adobe Systems Incorporated (http://www.adobe.com/), with Reserved Font Name 'Source'. All Rights Reserved. Source is a trademark of Adobe Systems Incorporated in the United States and/or other countries."
-    }
+    },
+    {
+      "name": "Source Code Pro",
+      "postScriptName": "SourceCodePro-BlackItalicItalic",
```
Comment (adrientetar) [inline ofl/sourcecodepro/METADATA.json]: Italic italic? Is this how the metadata is set up in the font?

**Comment by codeman38** [comment]
METADATA.json is definitely wrong here. The PS name in the font file itself, according to the ttx dump, is "SourceCodePro-BlackIt".

File: ofl/sourcecodepro/METADATA.json (hunk 1 above)
Comment (codeman38) [inline ofl/sourcecodepro/METADATA.json]: METADATA.json is definitely wrong here. The PS name in the font itself is `SourceCodePro-BlackIt`.

**Comment by futekov** [comment]
This PR has been pending for quite a long time and since user @mwberry doesn't seem to be very active recently I propose closing this request and considering this one from me - https://github.com/google/fonts/pull/166.

In addition to the [updated font files](https://github.com/adobe-fonts/source-code-pro/releases/tag/2.010R-ro/1.030R-it) I've made proper changes to the metadata info (both the JSON and PB files) and replaced the readme with the [official one from Adobe](http://www.adobe.com/products/type/font-information/source-code-pro-readme.html).

**Comment by davelab6** [comment]
Closing per @futekov 's suggestion :)

**Pull request closed without merging**